parcels/particle.py

```diff
@@ -356,26 +362,29 @@ def __init__(self, name, particleset, initial_dump=True):
         self.trajectory[:] = np.arange(particleset.size, dtype=np.int32)
 
         # Create time, lat, lon and z variables according to CF conventions:
-        self.time = self.dataset.createVariable("time", "f8", ("trajectory", "obs"), fill_value=0.)
+        self.time = self.dataset.createVariable("time", "f8", ("trajectory", "obs"), fill_value=-9999.)
```

Comment:
Arbitrary numerical fill values are rarely a good idea. Would `np.nan` not work?